modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/UserNameFields.es.js

```diff
@@ -216,7 +216,11 @@ class UserNameFields extends PortletBase {
 	 * @protected
 	 */
 	_removeLoadingIndicator() {
-		dom.exitDocument(this.one('#loadingUserNameFields'));
+		const loadingUserNameFields = this.one('#loadingUserNameFields');
+
+		if (loadingUserNameFields) {
```

Comment:
You may notice that I added few verifications before performing the `remove` because [exitDocument](https://github.com/metal/metal.js/blob/466812085dd29e37c90b835c64cb49555d76a294/packages/metal-dom/src/domNamed.js#L332) does that in the implementation.

Reply:
yeah... we could analyze this to see if that's even possible, that the element is not there. If it's guaranteed it'll be there, we wouldn't even need to check, but it's also okay, this is one of the reasons why we're trying to get away from this paradigm